spleeter_service/Dockerfile

```
@@ -0,0 +1,13 @@
+
+FROM python:3.9-slim
+
+WORKDIR /app
+
+# Install Spleeter and required dependencies
+RUN apt-get update && apt-get install -y ffmpeg && apt-get clean
+RUN pip install --no-cache-dir spleeter
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)